.gitignore

```diff
@@ -0,0 +1,30 @@
+# system files
+.DS_Store
+.ipynb_checkpoints
+venv
+.venv
+
+# folder
+temp/a
```

Comment:
Why not just `temp/`? do you only want to ignore the `temp/a` subdirectory?

Reply:
Another typo. How do I fix this?

Reply:
`.DS_Store` is a file created by VS code. Joanna has the same. It should me hidden, as far as I know.

Reply:
Yes, I usually add it to my ignore file